From e59ccfc3e0a27e2ef2dd0bed6376a9ed4637dc21 Mon Sep 17 00:00:00 2001
From: Even Xu <Feng.Xu@freescale.com>
Date: Wed, 30 Nov 2011 09:28:33 +0800
Subject: ENGR00163497-1 Add a "io_init" function to fsl camera platform data

Add a new member named "io_init" to fsl camera platform data, this
Function can initialize IO of camera sensor before it works.

Signed-off-by: Even Xu <b21019@freescale.com>
---
 include/linux/fsl_devices.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 1fa74c7..a0d44cb5 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -259,6 +259,7 @@ struct fsl_mxc_camera_platform_data {
 	u32 mclk;
 	u32 csi;
 	void (*pwdn)(int pwdn);
+	void (*io_init)(void);
 };
 
 struct mpc8xx_pcmcia_ops {
-- 
1.7.9.5

